packages/dataviews/src/style.scss

```diff
@@ -3,17 +3,19 @@
 	overflow: auto;
 	box-sizing: border-box;
 	scroll-padding-bottom: $grid-unit-80;
+	container: dataviews-wrapper / inline-size; /* stylelint-disable -- '@container' not globally permitted */
```

Comment:
I think that with this comment, all stylelint rules will be disabled in the code that follows. I think it is better to use `stylelint-disable-next-line` instead of `stylelint-disable` and further limit the rules to be disabled.

```scss
.dataviews-wrapper {
	/* stylelint-disable-next-line property-no-unknown -- '@container' not globally permitted */
	container: dataviews-wrapper / inline-size;
}

/* stylelint-disable-next-line scss/at-rule-no-unknown -- '@container' not globally permitted */
@container (max-width: 430px) {
}
```

Reply:
Oh you're right!